-------------------------------------------------------------------------------
Test set: Mobile_android_Test.Mobile_andoid_Test.AndroidInvokeAPPMobileTest
-------------------------------------------------------------------------------
Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 55.11 s <<< FAILURE! - in Mobile_android_Test.Mobile_andoid_Test.AndroidInvokeAPPMobileTest
test1(Mobile_android_Test.Mobile_andoid_Test.AndroidInvokeAPPMobileTest)  Time elapsed: 55.106 s  <<< ERROR!
org.openqa.selenium.WebDriverException: 
An unknown server-side error occurred while processing the command. Original error: 'app' option is required for reinstall
Build info: version: '3.12.0', revision: '7c6e0b3', time: '2018-05-08T14:04:26.12Z'
System info: host: 'U6017127-TPD-A', ip: '10.9.215.84', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_45'
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: 'app' option is required for reinstall
    at getResponseForW3CError (C:\Users\u6017127\AppData\Roaming\npm\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:800:13)
    at asyncHandler$ (C:\Users\u6017127\AppData\Roaming\npm\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:352:37)
    at tryCatch (C:\Users\u6017127\AppData\Roaming\npm\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\Users\u6017127\AppData\Roaming\npm\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (C:\Users\u6017127\AppData\Roaming\npm\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at GeneratorFunctionPrototype.invoke (C:\Users\u6017127\AppData\Roaming\npm\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:136:37)
    at <anonymous>
	at Mobile_android_Test.Mobile_andoid_Test.AndroidInvokeAPPMobileTest.setUp(AndroidInvokeAPPMobileTest.java:48)

test1(Mobile_android_Test.Mobile_andoid_Test.AndroidInvokeAPPMobileTest)  Time elapsed: 55.107 s  <<< ERROR!
java.lang.NullPointerException
	at Mobile_android_Test.Mobile_andoid_Test.AndroidInvokeAPPMobileTest.tearDown(AndroidInvokeAPPMobileTest.java:63)

